.eslintrc

```diff
@@ -7,7 +7,7 @@
         "no-debugger": 1
     },
     "parserOptions": {
-        "ecmaVersion": 6,
+        "ecmaVersion": 8,
```

Comment:
You might consider extending `https://github.com/pixijs/eslint-config-pixi`, which @fincn made to have a cohesive coding still between the main pixi.js repo.